### PR TITLE
Propose new FIP for Clio enhancements

### PR DESCRIPTION
**Pull request by adsorptionenthalpy**
Add FIP-0015 - CLIO Enhancements

### CODE REVIEW

**Review by pawmmm — CHANGES_REQUESTED** [review]
I suggest we add parameters to the commands and maybe even example entries and also reference any pull-requests in Implementation.

File: fip-0015.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+|system|sellram|
+|currency|retire|
+|currency|transfer|
+
+## Added functionality
+|Feature|Description|
+|---|---|
+|Rename keosd|keosd folder, binaries and symbols renamed fio-wallet to avoid confict with keosd used on systems with a shared EOS installation|
+|Default max_fee|max_fee is set to 800000000001LL so user does not have to define manually. Any required fee set by the producers under this value will process|
+|Custom Push Endpoints|Certain transactions are pushed to a FIO chain api push endpoint depending on the action. Any undefined go to /v1/chain/push_transaction|
```
Comment (pawmmm) [inline fip-0015.md:59]: 800000000001LL ?

Comment (adsorptionenthalpy) [reply fip-0015.md:59]: It was just the largest fee configured at the time + 1 SUF. Do you have a recommendation ?

Comment (adsorptionenthalpy) [reply fip-0015.md:59]: Corrected, removed LL

**Review by pawmmm — APPROVED** [review]

**Pull request merged**